app/src/main/kotlin/com/mapzen/erasermap/controller/MainActivity.kt

```diff
@@ -587,6 +585,10 @@ public class MainActivity : AppCompatActivity(), MainViewController, RouteCallba
     }
 
     override fun showReverseGeocodeFeature(features: List<Feature>) {
+        hideSearchResults()
+        centerOnCurrentFeature(features)
```

Comment:
Usually other mapping apps do not center the view on a reverse geo coded feature or a tapped poi feature. Do we need to center the map on these?